drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -172,6 +172,13 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
     block_settings_key = 'drag-and-drop-v2'
     has_score = True
 
+    def max_score(self):  # pylint: disable=no-self-use
+        """
+        Return the problem's max score, which for DnDv2 always equals 1.
```

Comment:
@nasthagiri I believe the same have already [been discussed](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/108#discussion_r85107013). DnDv2 calculates student grade as a [ratio of correct item placements to total count of items](https://github.com/open-craft/xblock-drag-and-drop-v2/blob/3ad5a184dfe6ecb6644603cc75973e19738a97da/drag_and_drop_v2/drag_and_drop_v2.py#L898), so max_score is one, but `grade = student_score * weight`.

We might consider making max_score equal to total number of items, and adjust grade calculations accordingly, but I'm concerned that this might be not backward compatible.

I'm not sure about this part:

> learners can get partial credit for some of the items they got correct.

If I'm not mistaking learners already receive partial credit if they haven't achieved 100% score.

Also, maybe there's a typo in [the formula](https://openedx.atlassian.net/wiki/display/TNL/Robust+Grades+Design#RobustGradesDesign-ProblemScores), but `weight` is in nominator and denominator, so it should cancel out, and the formula should become `learner_raw_score / max_score`.